app/controllers/parent/kids_controller.rb

```diff
@@ -1,21 +1,36 @@
 class Parent::KidsController < Parent::PortalController
 
+  def index
+  # List of all a parent's kids.
+  end
+
+  def show
+  # Details of specific kid.
```

Comment:
`show` and `index` will need instances of @kid and @kids, respectively, in order to render on the page.